Codegen/src/Generators/LinearNodeSourceCodeMapper.cs

```diff
@@ -263,6 +261,30 @@ internal class LinearCopyNode : LinearTokensNode
             public LinearCopyNode(CodeElement CodeElement) : base(CodeElement)
             {
             }
+
+            /// <summary>
+            /// Return the indented lines of copy node declaration
+            /// </summary>
+            public IEnumerable<ITextLine> GetCopyLines()
```

Comment:
At first I thought it was better to have a method instead of a property but here we can override the `Lines` property from `Node`. This would allow to keep `TypedDataNode.cs` untouched.

Reply:
Changed to property
https://github.com/TypeCobolTeam/TypeCobol/pull/1648/commits/c148777c31d93bcbf91b96ae92881f3f876a21d9